analysis/analysis.py

```diff
@@ -157,6 +156,47 @@ def compute_reread_counts(student_data, question, context):
 
     return organized_data
 
+def unique_word_pattern(student_data):
+    """
+    Take the CSV File and analyze the readers' responses based on the two different contexts of the
+    2 questions (this is an ad/this is just a short story); analyze if the total number of unique
+    responses changed as more and more readers' responses are analyzed. Eventually prints out the
+    pattern along with the number of unique word in the text.
+    :param student_data
```

Comment:
What sort of object is "student_data"?  Is it a str of a CSV file?  Is it a list of lists? etc.